examples/demo_app/config/environments/production.rb

```diff
@@ -22,8 +22,7 @@
   # Apache or NGINX already handles this.
   config.public_file_server.enabled = ENV['RAILS_SERVE_STATIC_FILES'].present?
 
-  # Compress JavaScripts and CSS.
-  config.assets.js_compressor = :uglifier
```

Comment:
Same as above re: `uglifier` not being needed since javascript is compiled externally.